angularFiles.js

```diff
@@ -189,21 +189,78 @@ var angularFiles = {
     'src/angular.bind.js'
   ],
 
-  'karmaModules': [
+  'karmaModulesBase': [
     'build/angular.js',
-    '@angularSrcModules',
+    'src/ngMock/*.js',
     'test/modules/no_bootstrap.js',
-    'test/helpers/*.js',
-    'test/ngAnimate/*.js',
-    'test/ngMessageFormat/*.js',
-    'test/ngMessages/*.js',
-    'test/ngMock/*.js',
-    'test/ngCookies/*.js',
-    'test/ngRoute/**/*.js',
-    'test/ngResource/*.js',
-    'test/ngSanitize/**/*.js',
-    'test/ngTouch/**/*.js',
-    'test/ngAria/*.js'
+    'test/helpers/matchers.js',
+    'test/helpers/privateMocks.js',
+    'test/helpers/support.js',
+    'test/helpers/testabilityPatch.js'
+  ],
+
+  'karmaModules-ngAnimate': [
+    '@karmaModulesBase',
+    '@angularSrcModuleNgAnimate',
+    'test/ngAnimate/**/*.js'
+  ],
+
+  'karmaModules-ngAria': [
+    '@karmaModulesBase',
+    '@angularSrcModuleNgAria',
+    'test/ngAria/**/*.js'
+  ],
+
+  'karmaModules-ngCookies': [
+    '@karmaModulesBase',
+    '@angularSrcModuleNgCookies',
+    'test/ngCookies/**/*.js'
+  ],
+
+  'karmaModules-ngMessageFormat': [
+    '@karmaModulesBase',
+    '@angularSrcModuleNgMessageFormat',
+    'test/ngMessageFormat/**/*.js'
+  ],
+
+  'karmaModules-ngMessages': [
+    '@karmaModulesBase',
+    'build/angular-animate.js',
```

Comment:
Hm... 🤔
Why is this necessary?

Reply:
Some tests in ngMessages are dependent on ngAnimate, e.g. https://github.com/angular/angular.js/blob/bb5a7e39ba2c8fe5afe1ec6f90ebedf22658a1c8/test/ngMessages/messagesSpec.js#L465

Reply:
Do we still need `ngAnimate` if we use `ngAnimateMock`?

Reply:
Theoretically yes, but I'm more comfortable with testing the actual animations instead of just the base provided by ng.